README.md

```diff
@@ -56,6 +56,7 @@ Clone or copy the source into your own project. There are sections marked with `
   * headers - added [helmet](https://www.npmjs.com/package/helmet)
   * CQRS - TODO
   * Request sanitisation - TODO
+  * cross-site scripting - added [csurf](https://github.com/expressjs/csurf) package which uses cookies to protect against [XSS](https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)), see [here](https://github.com/pillarjs/understanding-csrf) for how this works
```

Comment:
Can you increase the documentation so it's more detailed? I'm thinking for people who are just looking at this repo fresh and want to understand what this is for and why. I broke out the Configuration readme section in a new paragraph like [this](https://github.com/KainosSoftwareLtd/govuk_nodejs_typescript_example/pull/1/commits/2f33f4a47ec49a70b0c273d2b6f10ae259c5b8d3). Details like what you need to add to each form view to support this and how you handle whitelisting requests.